project/src/main/webapp/matchbrackets.js

```diff
@@ -0,0 +1,158 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
```

Comment:
if we have modified these files form the CodeMirror codebase, it may be useful to comment where we have changed things

Reply:
nothing has been modified, directly imported from codemirror